deploy/crds/kieapp.crd.yaml

```
@@ -670,120 +671,516 @@ spec:
-                                type: string
-                              jdbcURL:
-                                description: Database JDBC URL. For example, jdbc:mysql:mydb.example.com:3306/rhpam
-                                type: string
-                              maxPoolSize:
-                                description: Sets xa-pool/max-pool-size for the configured
-                                  datasource.
-                                type: string
-                              minPoolSize:
-                                description: Sets xa-pool/min-pool-size for the configured
-                                  datasource.
-                                type: string
-                              password:
-                                description: External database password
-                                format: password
-                                type: string
-                              username:
-                                description: External database username
-                                type: string
-                            required:
-                            - driver
-                            - jdbcURL
-                            - password
-                            - username
-                            type: object
-                          size:
-                            description: Size of the PersistentVolumeClaim to create.
-                              For example, 100Gi
+                          allowExternalFileRegister:
+                            description: Allow download of external (remote) files
+                              into runtime. Default value is false
+                            type: boolean
+                          compEnable:
+                            description: When set to true enables external components.
+                            type: boolean
+                          componentPartition:
+                            description: Components will be partitioned by the Runtime
+                              Model ID. Default value is true
+                            type: boolean
+                          configMapProps:
+                            description: Properties file with Dashbuilder configurations,
+                              if set, uniq properties will be appended and, if a property
+                              is set mode than once, the one from this property file
+                              will be used.
                             type: string
-                          storageClassName:
-                            description: The storageClassName to use for database
-                              pvc's.
+                          dataSetPartition:
+                            description: Datasets IDs will partitioned by the Runtime
+                              Model ID. Default value is true
+                            type: boolean
+                          enableBC:
```

same here as there is used enableBC and next property is called enableKieServer
Do not merger suggestion and property name change needs to be reflected in all places.
```suggestion
                          enableBusinessCentral:
```

---

updated.
thanks